pkg/kwokctl/utils/cmd_other.go

```diff
@@ -0,0 +1,52 @@
+//go:build !windows
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+func startProcess(ctx context.Context, name string, arg ...string) *exec.Cmd {
+	cmd := command(ctx, name, arg...)
+	cmd.SysProcAttr = &syscall.SysProcAttr{
+		// Setsid is used to detach the process from the parent (normally a shell)
+		Setsid: true,
```

Comment:
is there a reason we set the sysproc attribute in `startProcess` here where as in `cmd_windows.go` the sysproc attribute for hide window is done in `command`? could we make it consistent?

Reply:
This is the attribute I got after independent testing on both sides, the definition of the sysproc attribute of the two systems is different